extensions/card4l-eo/README.md

```diff
@@ -0,0 +1,184 @@
+| bbox            | **REQUIRED.** The bounding box of the acquisition.           | 1.4   |
+
+### STAC Item Properties
+
+#### CARD4L
+
+| Field Name                         | Data Type | Description                                                  | Req.  |
+| ---------------------------------- | --------- | ------------------------------------------------------------ | ----- |
+| card4l:specification               | string    | **REQUIRED.** The CARD4L specification implemented, either `SR` (Optical, Surface Reflectance) or `ST` (Optical, Surface Temperature). | *n/a* |
+| card4l:specification_version       | string    | **REQUIRED.** The CARD4L specification version. Currently always `5.0`. | *n/a* |
+| card4l:northern_geometric_accuracy | number    | An estimate of the northern geometric accuracy in meters.    | 1.8   |
+| card4l:eastern_geometric_accuracy  | number    | An estimate of the eastern geometric accuracy in meters.     | 1.8   |
+
+Note that all these fields here are aligned with the CARD4L SAR extension.
+
+#### Common Metadata
+
+| Field Name     | Description                                                  | Req.  |
+| -------------- | ------------------------------------------------------------ | ----- |
+| license        | Recommended to be specified in a STAC Collection.            | *n/a* |
+| datetime       | **REQUIRED.** The time of the acquisition, usually the central timestamp between `start_datetime` and `end_datetime`. | 1.3   |
+| start_datetime | Start time of the acquisition.                               | 1.3   |
+| end_datetime   | End time of the acquisition.                                 | 1.3   |
+| instruments    | **REQUIRED.**                                                | 1.9   |
+| constellation  | Constellation name in lower-case. Only if part of a constellation, e.g. `sentinel-2` for Sentinel 2A and 2B. Can often be derived from `platform`. | (1.9) |
+| platform       | Platform name in lower-case. Use a specific name such as `sentinel-2a` if part of constellation. MUST NOT duplicate `constellation`. | (1.9) |
+
+#### EO (Electro-Optical)
+
+| Field Name     | Description                                                  | Req.   |
+| -------------- | ------------------------------------------------------------ | ------ |
+| eo:cloud_cover |                                                              | 1.17   |
+| eo:bands       | You may include all bands specified in the assets again as an overview. | (1.10) |
+
+#### Processing
+
+| Field Name          | Description                                                  | Req.        |
+| ------------------- | ------------------------------------------------------------ | ----------- |
+| processing:facility |                                                              | 1.15        |
+| processing:software | String likely needs to be split into software name and version number. | 1.13 / 1.15 |
```

Comment:
This doesn't seem very specific, does it need to include software and version or is it more or less a free text field?

Reply:
It's 1:1 like in the main extension . I have not included data type and original description to avoid duplication

Reply:
see https://github.com/radiantearth/stac-spec/pull/933#discussion_r569345666